Application/EdFi.Ods.Features/ExternalCache/ExternalCacheProvider.cs

```diff
@@ -29,11 +27,9 @@ public class ExternalCacheProvider : IExternalCacheProvider
         private readonly TimeSpan _slidingExpiration;
         private readonly ILog _logger = LogManager.GetLogger(typeof(ExternalCacheProvider));
 
-        private static readonly JsonSerializerSettings _nonGenericSerializerSettings =
-            new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.Objects, ReferenceLoopHandling = ReferenceLoopHandling.Ignore };
-
-        private static readonly JsonSerializerSettings _customContractSerializerSettings =
-            new JsonSerializerSettings { ContractResolver = new CustomContractResolver(), ReferenceLoopHandling = ReferenceLoopHandling.Ignore };
+        // TypeNameHandling.None for https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2326
+        private static readonly JsonSerializerSettings _defaultSerializerSettings =
+            new JsonSerializerSettings { ReferenceLoopHandling = ReferenceLoopHandling.Ignore };
```

Comment:
Should we explicitly set TypeNameHandling.None instead of a comment?

Reply:
Updated